docs/build-on-linea/linea-version/index.mdx

```diff
@@ -11,6 +11,47 @@ import TokenBridgeL2 from "!!raw-loader!/files/testnet/2023-06-13-release/TokenB
 
 Find out all you need to know about the latest Linea versions.
 
+## Alpha v3 release notes
+
+### Summary
+
+The main objective of Alpha v3 is to implement EIP-4844 on Linea, following its introduction to 
+Ethereum mainnet with the Dencun upgrade on March 13.
+
+### Features
+
+#### March 26: EIP-4844
+
+Begin using blob data to post data to L1, significantly reducing data availability costs. 
+Blob data replaces `calldata` and is significantly cheaper.
```

Comment:
Begin using blobs to post compressed L2 data to L1, with the aim to reduce data availability costs. 
Blobs represents and alternative transient data storage to `calldata` and can be significantly cheaper in L1 ETH costs depending on market demand.